tests/cmd/e2e/main.go

```diff
@@ -188,22 +188,27 @@ func main() {
 		testBasic(&wg, cluster1)
 		testHostNetwork(&wg, cluster1)
 		oa.CheckDataRegionDisasterToleranceOrDie(cluster1)
+		oa.CleanTidbClusterOrDie(cluster1)
```

Comment:
```suggestion
		go oa.CleanTidbClusterOrDie(cluster1)
```

clean a cluster may need a few minutes. Can do it in a goroutine.

Reply:
it's already in the goroutine of this task (see previous code hidden by GitHub)

Reply:
what about deleting the `e2e` namespace at the end of main function and wait all resources are freed?

Reply:
Yes, but it increases the time of that goroutine. Failed to clean is not a failure of the whole e2e test.

Reply:
Just a suggestion, not a necessity.